lib/lookup.json

```diff
@@ -356,16 +361,20 @@
   "npm": {
     "maintainers": ["nodejs/npm"],
     "prefix": "v",
-    "skip": ["aix", "s390"]
+    "skip": ["aix", "s390", ">=17"],
+    "comment": "OpenSSL 3.0"
```

Comment:
skipping npm on node 17+ seems a bit extreme, regardless of the failure?

Reply:
If it fails everytime for a known reason, then it's just noise (there's an issue on npm/cli about it, I can add the link here).

Reply:
Sure, but why would any node >= 17 be released if it always fails tests for npm? Shouldn’t such a failure be a blocker for a node release?

Maybe the link will provide the context I’m missing here :-)

Reply:
The link is https://github.com/npm/cli/issues/3919

Reply:
So that means node 17 has been released multiple times, and will continue to be, without running npm’s own tests on it?

Reply:
Unfortunately, yes.

Reply:
oof, that seems like it would have been a pretty compelling reason to either wait on openssl 3 or wait on node 17 :-(

Reply:
> The link is npm/cli#3919

This is weird, as `pino` is also using `tap` and it doesn't have this issue in `citgm`.

The npm/cli#3919 issue only arises when using the fancy tap reporter, [`base`]( https://node-tap.org/docs/reporting/#base), and that reporter is only default when in a terminal where colors are enabled.

> The default when stdout is a terminal and colors are enabled.

My assumptions from the `pino` tests was that that the simpler [`tap`](https://node-tap.org/docs/reporting/#tap) reporter rather than the `base` one became picked for these tests.

> This is the default when stdout is not a terminal, or colors are disabled.

The `base` reporter is based on React + [Ink](https://www.npmjs.com/package/ink) and thus is a rather complex dependency for a reporter and not really what one would want to have tests use in `citgm`, as it adds a lot of possible failures on top of the actual tests.

Why is `pino` working in this regard, but not `npm`?

Reply:
> oof, that seems like it would have been a pretty compelling reason to either wait on openssl 3 or wait on node 17 :-(

FWIW that tradeoff was discussed in a [TSC meeting](https://youtu.be/XPRWSXKLdrY?t=408) - with the npm failure being explicitly discussed [at the end](https://www.youtube.com/watch?v=XPRWSXKLdrY&t=1539s).

Tangentially, npm was only added to CITGM in March this year, and has been a regularly failing since it landed for various reasons :/